packages/prg/src/index.spec.ts

```diff
@@ -81,22 +90,126 @@ describe("PrgAes128", () => {
     });
   });
 
-  it("performs rejection sampling correctly", async () => {
+  xit("performs rejection sampling correctly", async () => {
```

Comment:
Here's a new test case, using empty strings for the customization and binder inputs, PrgAes128, and Field64. (using a modified version of https://github.com/divergentdave/vdaf-rejection-sampling-search)

```
Rejection found, seed = Seed([0e, 10, 1c, 9b, 1b, b7, 96, ec, 4a, 87, 33, 1f, 71, cb, 8a, 82]), offset = 4 (length 5)
rejected = 18446744072109885969, next = 17851589531647539880
```

Reply:
added in c4283e3